docs/change.md

```diff
@@ -0,0 +1,168 @@
+record of how we arrived at such a consensus.
+
+### The Developers
+
+The core developers of PyScript often need to refactor or change code to ensure
+PyScript remains a high quality open source project. To this end, our changes
+will be:
+
+* Proposed via a discussion on GitHub, or debated in the community call,
+  or described on discord (or a mixture of all three). Our intention is to do
+  all our work in the open and with the full engagement of the community.
+* Delivered via a pull request, reviewed in public. The community is invited to
+  be a part of this review process (subscribe to these updates
+  via your GitHub account).
+* Described in our docs and release notes.
+
+We, as core maintainers, welcome constructive feedback about our processes.
+
+### External Tech
+
+The universe is always changing and PyScript needs to change with it. These
+changes happen in three ways:
+
+* **Dependencies**. The Python interpreters, upon which we depend, are active
+  open source projects who also make new releases. The core maintainers of
+  PyScript have excellent relationships with these upstream projects and we
+  will endeavour to ensure the latest changes appear in PyScript as soon as
+  possible,
+* **Security**. Inevitably some code we depend upon, or perhaps even
+  in PyScript itself, will encounter a security related problem. **PyScript
+  takes such problems very seriously** and will take all the necessary steps to
+  ensure security related changes are timely, well documented and taken with
+  the greatest of care and attention.
+* **Standards**. PyScript's parents are Python and the web. PyScript is itself
+  facilitated by web-assembly (WASM). All of these technical worlds rely on
+  standards to ensure compatibility. Such standards are evolving and changing
+  all the time and we (PyScript core developers) are actively involved in such
+  developments and will endeavour to ensure PyScript, when appropriate, is
+  updated to the latest versions of these standards.
+
+## Deprecation
+
+Sometimes we need to make breaking changes:
+
+* A new feature supercedes an old one.
+* An initial implementation of an API requires improvement.
+* A feature is no longer used and requires removal to save space / complexity.
+
+Breaking changes can cause disruption. Therefore we will be very careful when
+making such changes. Of primary importance are communication and
+predictability.
+
+### Communication
+
+Any breaking changes will be arrived at via the processes described in the
+[Change](#change) section.
+
+Since these will be done in public and with community engagement we hope the
+initial decision for a breaking change will come as no surprise. Furthermore,
+as the change is implemented there will be further opportunities for the change
+to be broadcast: via updates in our community call, via our upcoming PyScript
+newsletter, and in the formal technical discussions that take place on GitHub.
```

Comment:
I think these should be more explicitly described as part of the Deprecation process.